server/src/main/java/org/opensearch/index/store/FsDirectoryFactory.java

```diff
@@ -232,4 +239,33 @@ MMapDirectory getDelegate() {
             return delegate;
         }
     }
+
+    private static List<String> INDEX_STORE_HYBRID_ALL_EXTENSIONS = List.of(
```

Comment:
Can we define the defaults of nio and mmap as constants, then construct this list by combining those two lists? Something like:

```
Stream.concat(NIO_DEFAULTS.stream(), MMAP_DEFAULTS.stream()).collect(Collectors.toUnmodifiableList());
```

Reply:
I removed `INDEX_STORE_HYBRID_ALL_EXTENSIONS` and built the list directly when it's needed. When I tried to edit here it gave null exception. The setting defaults may not be defined out here?